src/main/kotlin/com/wutsi/blog/client/user/SearchUserRequest.kt

```diff
@@ -3,5 +3,6 @@ package com.wutsi.blog.client.user
 data class SearchUserRequest(
         val userIds: List<Long> = emptyList(),
         val limit: Int = 20,
-        val offset: Int = 0
+        val offset: Int = 0,
+        val blog: Boolean = false
```

Comment:
change this to 
```
val blog: Boolean? = null
```
If user don't set the `blog` value - the API should work as it works now.
But if `blog` value is set, then the API will filter using the blog criteria

Reply:
done